lib/client-assets.php

```diff
@@ -66,6 +66,28 @@ function gutenberg_get_script_polyfill( $tests ) {
 	return $polyfill;
 }
 
+/**
+ * Registers the main TinyMCE scripts.
+ */
+function register_tinymce_scripts() {
+	global $tinymce_version, $concatenate_scripts, $compress_scripts;
+	if ( ! isset( $concatenate_scripts ) ) {
+		script_concat_settings();
+	}
+	$suffix     = SCRIPT_DEBUG ? '' : '.min';
```

Comment:
Just wondering: is this intentional or are we meant to make it like this?

Reply:
It's intentional in the sense of this is mirroring exactly what's done on WP Core